test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -72,7 +73,8 @@ public class UtilityTests
 
             // Workaround annoying test failures such as "Expected sw.Elapsed >= TimeSpan.FromSeconds(2); Actual: 1999.4092" by waiting slightly less than 2 seconds
             // Not sure what causes it, but either Task.Delay sometimes doesn't wait quite long enough or there is some clock skew.
-            Assert.True(sw.Elapsed >= TimeSpan.FromMilliseconds(1990), $"Expected sw.Elapsed >= TimeSpan.FromMilliseconds(1990); Actual: {sw.Elapsed.TotalMilliseconds}");
+            TimeSpan roundedElapsedSpan = sw.Elapsed.RoundSeconds(digits: 1);
+            Assert.True(roundedElapsedSpan >= TimeSpan.FromMilliseconds(2), $"Expected roundedElapsedSpan >= TimeSpan.FromMilliseconds(2); Actual: {roundedElapsedSpan.TotalMilliseconds}");
```

Comment:
As written, this test won't pass. I think you needed to change the comparison to use FromSeconds.

Reply:
Actually the test will pass but its not testing what is intended :)

Reply:
Oops... yeah, rushed through this and completely missed that this was now using milliseconds. Updated.